libcxx/test/libcxx/atomics/atomics.syn/wait.pass.cpp

```diff
@@ -0,0 +1,33 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// UNSUPPORTED: c++03, c++11, c++14, c++17
+// UNSUPPORTED: no-threads
+// XFAIL: availability-synchronization_library-missing
+// XFAIL: !has-64-bit-atomics
+
```

Comment:
```suggestion

// This bug was first fixed in LLVM 19
// XFAIL: using-built-library-before-llvm-19
```

Reply:
I'm assuming that implies "Darwin"?

Reply:
https://buildkite.com/llvm-project/libcxx-ci/builds/35617#018fc382-8cf3-45dc-a8c8-c3c9d9ccd245

Unfortunately we have "unexpected passes" because of this line. It seems that the CI "Apple system" does have this feature 
```
using-built-library-before-llvm-19
```

and from what I can see, it is linking against the just built libc++ which has the fix in it
```
-L /Users/libcxx-buildkite-agent/libcxx.buildkite-agent/builds/f1-1-macminivault-com/llvm-project/libcxx-ci/build/apple-system/arm64/lib -lc++
```